asteroids_code/BoilerPlate/App.cpp

```diff
@@ -5,7 +5,7 @@
 #include "Palet.h"
 
 #include "Player.hpp"
-#include "Vector2.hpp"
+#include "Asteroids.hpp"
```

Comment:
Class names should be singular since they describe the behavior of a single object. Unless this is a helper class that handles lists.